### PR TITLE
Test depending on plasma-client #11

### PR DESCRIPTION
**Pull request by syuhei176**
https://github.com/syuhei176/plasma-rust-framework/blob/debug-target/client/Cargo.toml#L25

I wanna separate dependencies by target_os like below.
If target_os is android plasma-client use no-leveldb.
If target_os isn't android plasma-client use leveldb.
But when remove comment syntax, cargo always choose second dependency even if compile target is android.

```
[target.'cfg(target_os = "android")'.dependencies]
plasma-db = { path = "../db", default-features = false, features = ["no-leveldb"]}

# [target.'cfg(not(target_os = "android"))'.dependencies]
# plasma-db = {path = "../db"}
```

### CODE REVIEW

**Comment by omuomugin** [comment]
[MEMO]
most of the diff came from lock file.  (Just want to say diff it self is not that big)

**Comment by syuhei176** [comment]
This is answer. https://users.rust-lang.org/t/cargo-feature-specific-dependencies/27106

**Comment by omuomugin** [comment]
I see these error and cannot build it

```
error[E0658]: use of unstable library feature 'try_from' (see issue #33417)
```

File: rust/Cargo.toml [hunk 1]
```diff
@@ -8,6 +8,12 @@ edition = "2018"
 jni = { version = "0.10.2", default-features = false }
 plasma-core = { git = "https://github.com/cryptoeconomicslab/plasma-rust-framework.git" }
 
+[dependencies.plasma-client]
+git = "https://github.com/syuhei176/plasma-rust-framework.git"
```
Comment (omuomugin) [inline rust/Cargo.toml]: Why don't you create a repository under `cryptoeconomicslab` ? Just curious.

Comment (syuhei176) [reply rust/Cargo.toml]: I will change repository after https://github.com/cryptoeconomicslab/plasma-rust-framework/pull/118 is merged.

Comment (omuomugin) [reply rust/Cargo.toml]: Ah, I got you 👍

**Comment by syuhei176** [comment]
> I see these error and cannot build it
> 
> ```
> error[E0658]: use of unstable library feature 'try_from' (see issue #33417)
> ```

Try to run `rustup update`. I guess rust version is old.

File: rust/Cargo.toml (hunk 1 above)
Comment (syuhei176) [reply rust/Cargo.toml]: Updated👍

**Comment by omuomugin** [comment]
Sorry to be late, I succeeded to build app

> Try to run rustup update. I guess rust version is old.

As you said this was the cause.

**Comment by syuhei176** [comment]
Thanks merged!

**Pull request merged**